tests/PHPUnit/Integration/FrontControllerTest.php

```diff
@@ -83,6 +83,19 @@ public function test_authImplementationConfigured_EvenIfSessionAuthSucceeds()
         $this->assertEquals(Fixture::getTokenAuth(), $auth->getTokenAuth());
     }
 
+    public function test_ifHealthCheckParamPassedInUrl_ReturnsHealthCheckResponse(): void
```

Comment:
I know we have a lot test methods using `_` in their names, but if we want to aim PSR code style at some point, we should try to avoid that for new methods.

```suggestion
    public function testIfHealthCheckParamPassedInUrlReturnsHealthCheckResponse(): void
```

Reply:
Updated all my tests and updated other test names in the class